src/js/player.js

```diff
@@ -138,6 +138,9 @@ class Player extends Component {
     this.userActive_ = true;
     this.reportUserActivity();
     this.listenForUserActivity();
+
+    this.on('fullscreenchange', this.onFullscreenChange);
+    this.on('stageclick', this.onStageClick);
```

Comment:
Nevermind. Looks like we were just using it for `reportUserActivity` before, so I guess the player would need to listen for it.